spec/pagseguro/features/create_transaction_request_spec.rb

```diff
@@ -37,8 +37,8 @@
         status: [400, "Bad Request"], body: body, content_type: "text/xml"
     end
 
-    it "returns false" do
-      expect(transaction.create).to be_falsey
+    it "returns transaction" do
+      expect(transaction.create).to eq(transaction)
```

Comment:
@dmitryrck we can keep all these methods as `create` and return boolean. this way the api will be the same for every service like this. ok?